### PR TITLE
Frontend/updates

### PR DESCRIPTION
**Pull request by iamdey**

### CODE REVIEW

File: src/frontend/web_application/src/modules/message/models/Message.ts [hunk 1]
```diff
@@ -7,32 +7,53 @@ export class Message {
   constructor(props = {}) {
     Object.assign(this, props);
   }
+
```
Comment (MonsieurLanza) [inline src/frontend/web_application/src/modules/message/models/Message.ts:10]: Why those space ? Feels like LF/CRLF conversion ?

**Review by MonsieurLanza — APPROVED** [review]

File: src/frontend/web_application/src/modules/message/models/Message.ts (hunk 1 above)
Comment (iamdey) [reply src/frontend/web_application/src/modules/message/models/Message.ts:10]: probably prettier

**Comment by iamdey** [comment]
btw, this PR does not build due to ts errors

**Comment by iamdey** [comment]
build fixed,

replace raw redux configure store by reduxjs/toolkit

File: src/frontend/web_application/src/modules/message/models/Message.ts (hunk 1 above)
Comment (iamdey) [reply src/frontend/web_application/src/modules/message/models/Message.ts:10]: I remember, it's autofix eslint

**Comment by iamdey** [comment]
still something wrong at build, on live dev, the watcher restart in a loop, may be it's not blocking.

**Pull request merged**